code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm

```diff
@@ -109,3 +109,35 @@
 	desc = "Makes your nice clawed, scaled, hooved, armored, or otherwise just awfully calloused feet immune to glass shards."
 	cost = 1
 	var_changes = list("flags" = NO_MINOR_CUT) //Checked the flag is only used by shard stepping.
+
+/datum/trait/snowymarch //Oh my god those names are all awful.
+	name = "Snowy March"
+	desc = "Makes it easier to traverse snowy terrain."
+	cost = 1
+	var_changes = list("snow_movement" = -1)//Lower is less slowdown.
+
+/datum/trait/snowstride
+	name = "Snowy Stride"
+	desc = "It makes little difference to you whether the ground is snow or hard plating."
+	cost = 2
+	var_changes = list("snow_movement" = -2)//Lower is less slowdown.
+
+/datum/trait/puddlecrawl
+	name = "Puddle Crawl"
+	desc = "You know how not to drown. Mostly."
+	cost = 1
+	var_changes = list("water_movement" = -2)//Lower is less slowdown.
+
+/datum/trait/basinglide
+	name = "Basin Glide"
+	desc = "Skrell tier." //I have no idea for a description help
+	cost = 2
+	var_changes = list("water_movement" = -4)//Lower is less slowdown.
```

Comment:
Same as above but for water, and also really though put real descriptions for these please.

Reply:
By the way this trait absolutely helps a lot when in deeper water levels.